jina/helper.py

```diff
@@ -203,7 +203,7 @@ def parse_arg(v: str) -> Union[bool, int, str]:
     return v
 
 
-def countdown(t: int, reason: str = 'I am blocking this thread') -> None:
+def countdown(t: int, reason: str = 'I am blocking this thread'):
```

Comment:
why is it here removed?

Reply:
Apologies - turns out I misunderstood the recommended style for functions that have only an implicit return. (I thought you're supposed to omit the return, but in fact it's recommended to return None). I'll revert that change.